Podfile

```diff
@@ -77,7 +79,7 @@ end
 def shared_test_pods
   pod 'OHHTTPStubs/Swift', '~> 9.1.0'
   pod 'OCMock', '~> 3.4.3'
-  gutenberg_pods
+  gutenberg_pod
```

Comment:
Changed from pods to pod because the end goal is to have one single pod (which we do) and one single import.